src/Message/AbstractRequest.php

```diff
@@ -1096,6 +1103,24 @@ protected function buildPaymentMethod($paymentMethodType, $addAttributes = false
             if (!$this->isUpdate()) {
                 $paymentMethod->type = self::PAYMENT_METHOD_PAYPAL;
             }
+        } elseif ($paymentMethodType === self::PAYMENT_METHOD_APPLE_PAY && $card !== null) {
+            $applePay = new stdClass();
+
+            /**
+             * @var \Omnipay\Vindicia\NonStrippingCreditCard $card
+             */
+            $applePay->paymentInstrumentName = $card->getPaymentInstrumentName();
+            $applePay->paymentNetwork = $card->getPaymentNetwork();
+            $applePay->paymentData = $card->getToken();
+            $applePay->transactionIdentifier = $card->getTransactionIdentifier();
```

Comment:
looks like the name of this should be getApplePayTransactionReference (i think that was changed in a different PR at some point)

Reply:
could you change the names in the unit test to match that too?

Reply:
I think the unit tests were already changed

Reply:
@DanayaMel I mean stuff like TRANSACTION_IDENTIFIER and $this->transactionIdentifier in the unit test should be updated to match the new naming scheme